04-job-dsl-for-github-org/Dockerfile

```diff
@@ -1,9 +1,10 @@
 FROM jenkins/jenkins:2.71-alpine
 
 ENV JENKINS_REF /usr/share/jenkins/ref
+RUN mkdir -p $JENKINS_REF
 
 # install jenkins plugins
-COPY jenkins-home/plugins.txt $JENKINS_REF
+COPY jenkins-home/plugins.txt $JENKINS_REF/
 RUN /usr/local/bin/plugins.sh $JENKINS_REF/plugins.txt
```

Comment:
when `docker build` at the ubuntu  in `aliyun.com` , it build fail because the dir `/usr/share/jenkins/ref` does not existed .